src/components/Voting.tsx

```diff
@@ -116,9 +119,14 @@ export const Voting = () => {
           setModalOpen(ModalType.ERROR);
         }
       } catch (error) {
+        const errorString = String(error);
+        const truncatedError = errorString.length > 255 ? errorString.substring(0, 255) : errorString;
         console.error('Cast failed:', error);
-        if (error instanceof Error && address) {
-          track('Error', { message: error.message || 'Unknown error', address });
+        if (truncatedError && address) {
+          track('Error', {
+            address,
+            truncatedError,
+          });
```

Comment:
If `truncatedError` is more than 255 the track will also failed because it is adding the `address` (and maybe also the 'Error' word)

Is this how it works on Vercel?

Reply:
FYI: it worked in vercel, value itself has the limitation. But, we are moving to an api approach